android/src/main/kotlin/com/github/josxha/maplibre/MapLibreMapController.kt

```diff
@@ -82,11 +82,8 @@ class MapLibreMapController(
             val options =
                 MapLibreMapOptions
                     .createFromAttributes(context)
-                    .attributionEnabled(mapOptions.attribution)
-                    .logoEnabled(mapOptions.logo)
```

Comment:
Similar to the attribution on web, these options actually default to true. We need to disable them explicitly.